### PR TITLE
Update k8s "Get started" docs

### PR DESCRIPTION
**Pull request by pierwill**
Noticed some easy improvements to the Kubernetes section of the InfluxDB "Get started" doc:

- Add instructions for using kind
- Bump version to 2.0.3

---

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/get-started.md [hunk 1]
```diff
@@ -323,15 +323,20 @@ docker exec -it influxdb /bin/bash
 
 ### Install InfluxDB in a Kubernetes cluster
 
-The instructions below use Minikube, but the steps should be similar in any Kubernetes cluster.
+The instructions below use minikube or kind, but the steps should be similar in any Kubernetes cluster.
```
Comment (sanderson) [inline content/influxdb/v2.0/get-started.md:326]:
```suggestion
The instructions below use **minikube** or **kind**, but the steps should be similar in any Kubernetes cluster.
```

File: static/downloads/influxdb-k8-minikube.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ spec:
                 app: influxdb
         spec:
             containers:
-              - image: quay.io/influxdb/influxdb:v2.0.1
+              - image: quay.io/influxdb/influxdb:v2.0.3
```
Comment (sanderson) [inline static/downloads/influxdb-k8-minikube.yaml:26]: We'll need to be sure to update this once 2.0.4 goes out. Is there a `latest` tag we could use so we don't have to remember to update this on each release?

Comment (pierwill) [reply static/downloads/influxdb-k8-minikube.yaml:26]: Does not appear to be a `latest` atm: https://quay.io/repository/influxdb/influxdb?tag=v2.0.3&tab=tags.

Comment (sanderson) [reply static/downloads/influxdb-k8-minikube.yaml:26]: K, will you update this on the 204-changelog branch for the 2.0.4 release?

**Review by sanderson — APPROVED** [review]

**Pull request merged**